src/components/top_bar/top_bar.xml

```diff
@@ -41,15 +41,15 @@
           <div t-else="" class="o-toolbar-tools">
             <div
               class="o-tool"
-              title="Undo"
+              title="Undo (Ctrl+Z)"
```

Comment:
do not add the keybinds in the title

Reply:
I thought that the task was about, Hard copying the toolips from Gsheet. Which is why I added keybinds to the title. Should I remove from all places or just this one?

Reply:
It's not what I had in mind but you are right, it's better this way